app/card_content.tsx

```diff
@@ -0,0 +1,74 @@
+interface ImageDetails {
+    title: string,
+    path: string,
+}
+
+interface Details {
+    title: string,
+    text: string,
+}
+
+interface StageLayoutCardContent {
+    image: ImageDetails,
+    summary: Details,
+    description: Details,
+    stack: Details,
+}
+
+interface StageCardContent {
+    projectName: string,
+    date: string,
+    content: StageLayoutCardContent
+}
+
+
+const cardContent: { projects: StageCardContent[] } = {
+    projects: [
+        {
+            projectName: "Highlighted project",
```

Comment:
nit: `projectName` is redundant since this is in the projects list. 

```suggestion
            name: "Highlighted project",
```